packages/angular_devkit/build_angular/src/utils/i18n-options.ts

```diff
@@ -249,10 +249,35 @@ export async function configureI18nBuild<T extends BrowserBuilderSchema | Server
     buildOptions.outputPath = tempPath;
 
     // Remove temporary directory used for i18n processing
-    process.on('exit', () => {
+    const deleteTempDirectory = () => {
```

Comment:
Nit: Can this be moved to the top-level scope (accept `tempPath` as a parameter)? Does it need to be in this closure?